asciidoctorj-core/src/main/java/org/asciidoctor/extension/Processor.java

```diff
@@ -40,42 +40,59 @@
      * </verbatim>
      * </pre>
      * </p>
+     *
+     * @deprecated Use the annotation {@link ContentModel} instead to describe this Processor
      */
+    @Deprecated
     public static final String CONTENT_MODEL = "content_model";
```

Comment:
Totally right.
I was probably thinking that the whole manual configuration should be deprecated and happen via annotations where also deprecating this made sense.
I will "undeprecate" it again :-)